mipengine/algorithms/anova.py

```diff
@@ -20,7 +29,46 @@ class AnovaResult(BaseModel):
     f_pvalue: List[Optional[float]]
 
 
-class AnovaTwoWay(Algorithm, algname="anova"):
+class AnovaTwoWay(Algorithm, algname=ALGORITHM_NAME):
+    @staticmethod
+    def get_specification():
+        return AlgorithmSpecification(
+            name=ALGORITHM_NAME,
```

Comment:
The algorithm name is already available in the classvar `self.algname`, so no need for a global constant.